roxie/ccd/ccdserver.cpp

```diff
@@ -5013,7 +5045,18 @@ class CRemoteResultAdaptor : implements IEngineRowStream, implements IFinalRoxie
         mu.clear();
         SimpleActivityTimer t(unpackerWaitCycles, timeActivities);
         unsigned ctxTraceLevel = activity.queryLogCtx().queryTraceLevel();
-        unsigned timeout = remoteId.isSLAPriority() ? slaTimeout : (remoteId.isHighPriority() ? highTimeout : lowTimeout);
+
+        unsigned timeout = lowTimeout;
```

Comment:
I still need to adjust this timeout if we have lowered priority after some elapsed time.